Source/ASDisplayNode.mm

```diff
@@ -453,6 +453,11 @@ - (void)asyncTraitCollectionDidChangeWithPreviousTraitCollection:(ASPrimitiveTra
           CGFloat cornerRadius = _cornerRadius;
           ASCornerRoundingType cornerRoundingType = _cornerRoundingType;
           UIColor *backgroundColor = _backgroundColor;
+          if (@available(iOS 13.0, *)) {
+             UITraitCollection *tempTraitCollection = [UITraitCollection traitCollectionWithUserInterfaceStyle:_primitiveTraitCollection.userInterfaceStyle];
+             backgroundColor = [backgroundColor resolvedColorWithTraitCollection:tempTraitCollection];
```

Comment:
Might be a stupid question, but why can't we resolve the color using `_primitiveTraitCollection` here? What makes it different from a new one created from its user interface style and why?

Reply:
resolvedColorWithTraitCollection need a UITraitCollection class